volume/core.go

```diff
@@ -20,6 +20,16 @@ import (
 	"github.com/puppetlabs/wash/plugin"
 )
 
+// Interface presents methods to access the volume.
+type Interface interface {
```

Comment:
This is a lot better. Only suggestions I have are:
* `Interface` => `Volume`
* `VolumeList` => `ListDir`
* `VolumeRead` => `ReadFile` (or `OpenFile` to keep it consistent w/ `Readable#Open`)
* `MakeEntries` => `List`

Reply:
Then it would be `volume.Volume`. `<package>.Interface` is not uncommon in the go standard library. I'm open to other suggestions, but not sure `volume.Volume` is an improvement.

I went with VolumeList and VolumeRead to namespace them since we're adding on to the Entry interface.